### DISCUSSION TITLE
Welcome to BiocChallenges Discussions!

### DISCUSSION THREAD

**Discussion by kevinrue** [kevinrue/BiocChallenges2#24] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LTLA** [comment]
first!

**Reply by federicomarini** [reply]
First! (0-based)

**Reply by alanocallaghan** [reply]
Its spelled "frist"

**Comment by vjcitn** [comment]
it gets #24 because it has something to do with the sequence of issues?

**Reply by alanocallaghan** [reply]
Github doesn't distinguish between repo elements when it comes to numbering I guess
